FirstTask.py

```diff
@@ -0,0 +1,230 @@
+            return True
+        else:
+            return False
+    
+    def draw_money(self, money):
+        if self.balance >= money:
+            self.balance -= money
+            return True
+        else:
+            return False
+    
+    def account_details(self):
+        print("Account # ", self.account_no)
+        print("Name: ", self.name)
+        print("Balance: ", self.balance)
+        print("Account Type: ", self.account_type)
+  
+class Bank:
+
+    bank_accounts = []
+    logged_in = False
+
+    def total_account(self):
+        return (len(self.bank_accounts))
+
+    def create_account(self, name, code, account_type):
+        self.bank_accounts.append(BankAccount(name, code, 1000+len(self.bank_accounts)+1, 0, account_type))
+
+    def find_account(self, account_number):
+        for i in range(len(Bank.bank_accounts)):
+            if account_number == Bank.bank_accounts[i].account_number():
+                return True, Bank.bank_accounts[i]
+        return False, "Account not found!"
+
+    def login(self, account_number, code):
+        status, response = self.find_account(account_number)
+        if status:
+            if code == response.pass_code():
+                self.logged_in_account = response
+                Bank.logged_in = True
+                return True, response
+        return False, "Account Not Found!"
+
+    def transfer_money(self, sender_account_number, receiver_account_number, amount):
+        status_1, response_1 = self.find_account(sender_account_number)
+        status_2, response_2 = self.find_account(receiver_account_number)
+        if status_1 and status_2:
+            if response_1.draw_money(amount):
+                response_2.deposit_bill(amount)
+                return True
+        else:
+            return False
+    
+    def compare(self, current_account, other_account):
+        status_1, response_1 = self.find_account(current_account)
+        status_2, response_2 = self.find_account(other_account)
+        if status_1 and status_2:
+            if response_1.check_balance() > response_2.check_balance():
+                return True, 1
+            if response_1.check_balance() < response_2.check_balance():
+                return True, -1
+            if response_1.check_balance() == response_2.check_balance():
+                return True, 0
+        else:
+            return False, f"{other_account} Account Not Found!"
+    
+   
+    def account_details(self, account_number):
+        status,response = self.find_account(account_number)
+        if status:
+            response.account_details()
+        else:
+            return False
+   
+    def logout(self):
+        self.logged_in = None
+        return self.logged_in
+
+    def is_logged_in(self):
+        if self.logged_in:
+            return True
+        return False
+
+def run():
+    account_number_id = 1000 # Wrirting here just to print account number for user after account creation 

```

Comment:
Why do we need to manage it separately when we can just show the `account_details` after the account creation? The purpose of moving the account number logic in `Bank` class was to not manage it here.